### DISCUSSION TITLE
Can rlGetCompressedFormatName  be rafactored?

### DISCUSSION THREAD

**Discussion by RicoP** [raysan5/raylib#2105] [Ideas]
https://github.com/raysan5/raylib/blob/2021f09db4df72aea9f963666435bddf0718df94/src/rlgl.h#L4383

I think rlGetCompressedFormatName could be refactored to be just a sequence of returns, like in rlGetPixelFormatName. 

Calling rlGetCompressedFormatName twice in a row will also alter the buffer of the first caller.

**Comment by raysan5** [comment]
Yes, you are right. Reviewed.
